### PR TITLE
Integrate admin APIs for people and confirmation code

### PR DESCRIPTION
**Pull request by jgbaylon**
Goal: To create API clients for people and confirmation code
- added client methods for people api
- added client methods for confirmation code api
- renamed PeopleDomain to People
- some cleanup

Tickets:
https://www.pivotaltracker.com/story/show/131825775
https://www.pivotaltracker.com/story/show/132187567

@cirelee @johnswekenyi @alan10FMe

### CODE REVIEW

File: src/test/java/co/tala/appium/client/PeopleApiClient.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    public static final String NATIONAL_ID = "nid";
+    public static final String NATIONAL_ID_DATE = "nid_date";
+    public static final String SEX = "sex";
+    public static final String COUNTRY = "country";
+    public static final String REFERRER = "referrer";
+    public static final String SIGNUP_DATE = "signup_date";
+    public static final String SIGNUP_SOURCE = "signup_source";
+    public static final String SIGNUP_SOURCE_TYPE = "signup_source_type";
+
+    private static final String PATH_PEOPLE = "/api/People";
```
Comment (cirelee) [inline src/test/java/co/tala/appium/client/PeopleApiClient.java]: PATH_API_PEOPLE

File: src/test/java/co/tala/appium/client/PeopleApiClient.java [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                .put(REFERRER, people.getReferrer())
+                .put(SIGNUP_DATE, DateUtils.toSqlTimestamp(people.getSignUpDate()))
+                .put(SIGNUP_SOURCE, SignUpSource.toName(people.getSignUpSource()))
+                .put(SIGNUP_SOURCE_TYPE, people.getSignUpSourceType())
+                .toString();
+    }
+
+    private People jsonObjectToPeople(JSONObject jsonObject) {
+        return People.builder()
+                .id(jsonObject.isNull(ID) ? null : jsonObject.getInt(ID))
```
Comment (cirelee) [inline src/test/java/co/tala/appium/client/PeopleApiClient.java]: To make this more readable, you can create util functions which does the null check and return null or non-null:

 - private String jsonGetString(JSON Object jsonObject, String jsonName)
 - private Int jsonGetInt(JSON Object jsonObject, String jsonName)
 - private Long jsonGetLong(JSON Object jsonObject, String jsonName)

File: src/test/java/co/tala/appium/client/PeopleApiClient.java (hunk 1 above)
Comment (jgbaylon) [inline src/test/java/co/tala/appium/client/PeopleApiClient.java]: renamed

**Comment by cirelee** [comment]
LGTM

**Pull request merged**